src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -66,26 +66,26 @@ export const FeatureFeaturedLink: React.FC<FeatureFeaturedLinkProps> = ({
             style={{ backgroundColor: color("black10") }}
           />
 
-          <Title size="6" color="white100" p={2} pt={9} unstable_trackIn>
+          <Title variant="title" color="white100" p={2} pt={9}>
```

Comment:
I am really digging these new typography components

Reply:
`Title variant="title"` is a bit confusing -- maybe do `.attrs({ variant: "tittle" })` up at the definition?